x-pack/plugins/index_management/__jest__/client_integration/index_template_wizard/template_edit.test.tsx

```diff
@@ -98,6 +98,60 @@ describe('<TemplateEdit />', () => {
 
       expect(find('fieldsListItem').length).toBe(1);
     });
+
+    it('allows you to save the template', async () => {
```

Comment:
Nit: can we clarify that this is an untouched or unmodified template in this description? Just in case someone edits this and thinks it's a good idea to improve the tests by populating the template with some input.